RELEASES.md

```diff
@@ -33,37 +47,54 @@ We leverage `pip` as a convient way to install pre-build libraries and tools. Th
 > If you _really_ want a system-wide install, you can pass `--break-system-packages` to `pip` outside a virtual enivornment.
 > In this case, commandline interface shims for executables are installed to `/usr/local/bin`, which normally has precedence over `/usr/bin` and might therefore conflict with a previous installation of ROCm.
 
-To install TheRock from pip, you must provide a link to the index page for your desired GPU architecture.
-One of the following commands can be used to install `rocm-sdk[libraries,devel]`.
+To install TheRock from pip, you must provide a link to the index page for your
+desired GPU architecture. One of the following commands can be used to install
+`rocm[libraries,devel]`:
```

Comment:
Add a "please check back regularly as these instructions will change as we migrate to official indexes and protect layout" or similar

Reply:
Done.